### ISSUE TITLE
putItem { ReturnValues: "UPDATED_OLD" } issue

### ISSUE THREAD

**Issue by databank** [mhart/dynalite#56]
Hi,

first of all, thank you for this npm module, 

Im using dynalite for "npm test" and I'm seeing differences between AWS and dynalite when using
putItem and "ADD"

# dynalite
```
putItem {
	"TableName": "test_hash_range",
	"Item": {
		"hash": {
			"S": "test-updated-old"
		},
		"range": {
			"N": "1"
		},
		"number": {
			"N": "10"
		}
	},
}
putItem {
	"TableName": "test_hash_range",
	"Key": {
		"hash": {
			"S": "test-updated-old"
		},
		"range": {
			"N": "1"
		}
	},
	"AttributeUpdates": {
		"number": {
			"Action": "ADD",
			"Value": {
				"N": "20"
			}
		}
	},
	"Expected": {
		"hash": {
			"Exists": true,
			"Value": {
				"S": "test-updated-old"
			}
		},
		"range": {
			"Exists": true,
			"Value": {
				"N": "1"
			}
		}
	},
	"ReturnValues": "UPDATED_OLD"
}
```
returns
```
{ number: 30 }
```

# AWS
```
putItem {
	"TableName": "test_hash_range",
	"Item": {
		"hash": {
			"S": "test-updated-old"
		},
		"range": {
			"N": "1"
		},
		"number": {
			"N": "10"
		}
	},
}
putItem {
	"TableName": "test_hash_range",
	"Key": {
		"hash": {
			"S": "test-updated-old"
		},
		"range": {
			"N": "1"
		}
	},
	"AttributeUpdates": {
		"number": {
			"Action": "ADD",
			"Value": {
				"N": "20"
			}
		}
	},
	"Expected": {
		"hash": {
			"Exists": true,
			"Value": {
				"S": "test-updated-old"
			}
		},
		"range": {
			"Exists": true,
			"Value": {
				"N": "1"
			}
		}
	},
	"ReturnValues": "UPDATED_OLD"
}
```
returns
```
{ number: 10 }
```

**Comment by mhart** [comment]
Hmm, are you sure you're using `PutItem`? The API doesn't allow you to update attributes with `PutItem`, it just replaces the whole object: http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_PutItem.html

Is this `UpdateItem` perhaps? I'll look into it

**Comment by mhart** [comment]
Found it! It was a problem in `UpdateItem` – thanks for spotting this. I've fixed it in v1.0.1

**Comment by databank** [comment]
yes, updateItem sorry, thanks for the fix!
